rust/yarp-sys/tests/parser_tests.rs

```diff
@@ -0,0 +1,212 @@
+use std::{
+    cell::OnceCell,
```

Comment:
I've removed the 1.59.0 MSRV (which was only initially specified because of a `bindgen` call that generates code that's dependent on 1.59); I removed that code and the MSRV.